cs_insights_prediction_endpoint/models/model_user.py

```diff
@@ -16,27 +16,27 @@ class UserModel(BaseModel):
         email (str): email of the user
         password (str): password of the user
         fullname (Optional[str]): fullname of the user
-        isAdmin (Optional[bool]): flag indicating whether it is a admin
-        isActive (Optional[bool]): flag indicating whether the user is still active
+        is_admin (Optional[bool]): flag indicating whether it is a admin
+        is_active (Optional[bool]): flag indicating whether the user is still active
     """
 
     email: str = Field(...)
     fullname: Optional[str] = Field()
-    isAdmin: Optional[bool] = Field()
-    isActive: Optional[bool] = Field()
+    is_admin: Optional[bool] = Field()
+    is_active: Optional[bool] = Field()
 
-    # XXX-TN change isAdmin to groups
-    # TODO-TN isActive is currently getting ignored
+    # XXX-TN change is_admin to groups
+    # TODO-TN is_active is currently getting ignored
 
     class Config:
         """Configuration for UserModel"""
 
         schema_extra = {
             "example": {
                 "email": "admin@nlp.de",
-                "fullname=": "admin",
+                "fullname": "admin",
                 "token": None,
-                "isAdmin": True,
```

Comment:
We need to take extra care here. If this is the user model of the backend, we need camelCase because it returns the fields exactly like in the backend. Please check that.